py3status (3.5-1) unstable; urgency=low

  * Autogenerated by py2dsp v1.20151008 (Closes: #780806)

 -- Sebastien Delafond <seb@debian.org>  Sat, 05 Aug 2017 20:27:48 +0000

